.travis.yml

```diff
@@ -62,6 +68,21 @@ base_install: &base_install |-
 
   [ "$TRAVIS_PULL_REQUEST" != "false" ] || export PYTEST_SENTRY_ALWAYS_REPORT=1
 
+install_volta: &install_volta |-
+  command -v volta && exit 0
+  wget --quiet "https://github.com/volta-cli/volta/releases/download/v$VOLTA_VERSION/volta-$VOLTA_VERSION-linux-openssl-1.0.tar.gz"
+  tar -xzf "volta-$VOLTA_VERSION-linux-openssl-1.0.tar.gz" -C "${HOME}/bin"
+  # Running `volta -v` triggers setting up the shims in VOLTA_HOME (otherwise node won't work)
+  volta -v
+
+install_node_dependencies: &install_node_dependencies
+  # Running `node -v` and `yarn -v` triggers Volta to install the versions set in the project.
+  # There might be a more readable `volta install` in the future that would replace this magic.
+  # Tracking: https://github.com/volta-cli/volta/issues/653#issuecomment-628909923
+  node -v
+  yarn -v
+  yarn install --frozen-lockfile
```

Comment:
FWIW, if you don't want / need the version output, you shouldn't need to run `node -v` and `yarn -v` separately. Running the `yarn install` command will cause Volta to detect the missing node & yarn and automatically download them.

Reply:
I think the separation may help caching that said not sure if that's true in TravisCI (def true in Docker).

Reply:
Hmm, it might be useful to have the version output though. @BYK what do you think?

Reply:
>Hmm, it might be useful to have the version output though. @BYK what do you think?

I'm mostly indifferent as if the version doesn't match, we have way bigger problems. The only scenario I see this being useful is using a `nodejs` base image and `volta` failing somewhere so we end up using the system-default `yarn` and don't realize it and spend hours debugging this 🙂 So I guess, let's keep it? (this would also allow us to measure yarn install time accurately if we want to in the future as otherwise this will be bundled with `node` and `yarn` installation times)

Reply:
> this would also allow us to measure yarn install time accurately if we want to in the future as otherwise this will be bundled with node and yarn installation times

That's the clincher for me. Thanks for the suggestion @charlespierce, but we're going to stick with this way right now. A `volta install` would make the most sense here.

Reply:
:+1: Makes sense